torch/testing/_internal/common_device_type.py

```diff
@@ -1008,6 +1008,8 @@ class OpDTypes(Enum):
     torch.int8,
     torch.uint8,
     torch.bool,
+    torch.float8_e4m3fn,
+    torch.float8_e5m2,
```

Comment:
Why this change is necessary?

Reply:
Without this change, all UTs in test_ops.py will fail as follows. `supported` is {torch.float8_e4m3fn}, and `ANY_DTYPE_ORDER` do not contain fp8 data types. I didn't combine dtypes in CUDA and CPU, because CUDA only tests float8_e4m3fn, and CPU tests float8_e4m3fn and float8_e5m2.

```
Traceback (most recent call last):
  File "/home/yanbingj/projects/pytorch/torch/testing/_internal/common_device_type.py", line 1125, in _parametrize_test
    next(dtype for dtype in ANY_DTYPE_ORDER if dtype in supported)
StopIteration

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/yanbingj/projects/pytorch/test/test_ops.py", line 2840, in <module>
    instantiate_device_type_tests(TestCommon, globals())
  File "/home/yanbingj/projects/pytorch/torch/testing/_internal/common_device_type.py", line 934, in instantiate_device_type_tests
    device_type_test_class.instantiate_test(
  File "/home/yanbingj/projects/pytorch/torch/testing/_internal/common_device_type.py", line 515, in instantiate_test
    for (
RuntimeError: generator raised StopIteration
```